core/silkworm/consensus/engine_test.cpp

```diff
@@ -61,6 +63,12 @@ TEST_CASE("Consensus Engine Seal") {
     CHECK(consensus_engine->validate_seal(fake_header) == ValidationResult::kOk);
 }
 
+TEST_CASE("Consensus Engine Clique") {
+    BlockHeader fake_header{};
+    auto consensus_engine = engine_factory(kGoerliConfig);  // Clique consensus engine
+    CHECK(consensus_engine->validate_seal(fake_header) == ValidationResult::kOk);
+}
+
```

Comment:
I would move this test case to a `engine_test.cpp` module dedicated to Clique consensus

Reply:
ok